src/main/java/babysitter/BabySitter.java

```diff
@@ -18,22 +19,19 @@ public BabySitter(int startTime, int endTime, int bedTime) {
 		this.bedTime = convertAm(bedTime);
 	}
 
-	boolean isValidWorkHours(int startTime, int endTime) {
+	boolean isValidWorkHours() {
```

Comment:
Passing these arguments would cause this method to fail if the original end time was after midnight (because it wouldn't have been converted yet). Instead, don't take any arguments, and use the instance variables from the constructor instead.